packages/react-components/react-positioning/etc/react-positioning.api.md

```diff
@@ -28,9 +28,9 @@ export function createArrowStyles(options: CreateArrowStylesOptions): GriffelSty
 // @internal
 export type CreateArrowStylesOptions = {
     arrowHeight: number | undefined;
-    borderWidth?: GriffelStyle['borderBottomWidth'];
```

Comment:
with `griffel` bump

`borderBottomWidth` -> `export type BorderBottomWidth<TLength = (string & {}) | 0> = Globals | DataType.LineWidth<TLength>;`


whilst borderBottom shorthand overload (`...shorthands.borderBottom(borderWidth, borderStyle, borderColor),`)  uses for `borderWidth` -> `BorderWidthInput` ->  `export type BorderWidth<TLength = (string & {}) | 0> = Globals | DataType.LineWidth<TLength> | (string & {});`